scripts/newinstall.sh

```diff
@@ -217,6 +214,43 @@ if true; then
 	fi
 fi
 
+##########	Test/warn about the existence of ~/.eups
+
+if true; then
+	if [[ -e ~/.eups ]]; then
+		TIMESTAMP=$(date +%Y%m%d%S)
+		cat <<-EOF
+
+		Problems have been observed when re-installing or upgrading EUPS while
+		preserving a pre-existing ~/.eups directory.  This directory primarily
+		contains cache data and is safe to remove when EUPS is not actively
```

Comment:
You should definitely not remove ~/.eups/startup.py and almost certainly not remove ~/.eups/manifest.remap (if present).  

The other files are not just caches --- they were invented and named by NCSA.  In particular, that's where user tags are defined.   I would only delete them as a last resort.